__mocks__/next/image.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
 import type { ImageProps } from 'next/image'
+import _ from 'lodash'
 
-const Image = (props: ImageProps) => <img {...props} />
+const Image = (props: ImageProps) => {
+  //not all Image properties are allowed in <img/> tag
```

Comment:
could you add comment on why `objectFit` is important here so we don't accidentally remove it?

Example:

```
// ObjectFit causes warning, but we need to add it because curriculum page images won't fill up its allocated space and looks weird without it.
```

This way we won't regress anymore

Reply:
I updated comment. Just to clarify `objectFit` doesn't cause any errors in production, it's a completely valid [property](https://nextjs.org/docs/api-reference/next/image#objectfit) for `next/Image`.